spec/Appendix B -- Grammar Summary.md

```diff
@@ -322,6 +322,7 @@ ExecutableDirectiveLocation : one of
   `FRAGMENT_SPREAD`
   `INLINE_FRAGMENT`
   `VARIABLE_DEFINITION`
+  `OBJECT_FIELD`
```

Comment:
So that changes the meaning of `FIELD` location. Does new location make sense?

Reply:
I don't see how it changes anything about the FIELD location. FIELD and OBJECT_FIELD are unrelated. But I have a feeling I'm missing your point. Can you say more?

Reply:
What is the difference between FIELD and OBJECT_FIELD ?

Reply:
FIELD refers to a GraphQL field (see section 2.5 of the spec). OBJECT_FIELD refers to a GraphQL ObjectField (2.9.8). They have nothing to do with each other. If you look at the proposed grammar change, the directive is supposed before the Name element of the ObjectField.

Reply:
To avoid this confusion and make the spec easier to understand, it would be helpful to refer to the section in the spec where each of these items refer to. I will make a separate PR for that.

Reply:
OK. Now I understand. At first I thought about something completely different.